docs/general/kusama/kusama-statemine.md

```diff
@@ -0,0 +1,86 @@
+
+:::
+
+To create an asset on Statemine, you would need a deposit of 1 KSM. Ensure that your Statemine account balance more than 1 KSM,
+which should account for the transdaction fee as well. You can send KSM from a Kusama account to a statemine account using the teleport functionality. For instructions on teleporting KSM, check this [tutorial on Teleports](../../learn/learn-teleport).
+
+Assuming you have the required KSM balance on your Statemine account, the following instructions should let you successfully create an asset on Statemine
+
+- Access Statemine through Polkdot-JS Apps UI.
+- Navigate to Network > Assets.
+
+![Navigate to Assets page](../../assets/kusama/statemine-asset-0.png)
+
+- Click on the create button and you will be presented with a pop-up window. Choose the creator account, name of the asset to be
+displayed on Statemine, the asset's symbol, number of decimals for the asset, the minimum balance required to hold this asset on a Statemine account and the most important field of your asset - the unique asset ID. The UI would not let you enter an ID that has already been taken. After all the details are entered, click on the next button.
+
+![Add Asset Metadata](../../assets/kusama/statemine-asset-1.png)
+
+- Choose the admin, issuer and the freezer accounts for your asset and click on the create button.
+
+![Asset managing accounts](../../assets/kusama/statemine-asset-2.png)
+
+- Sign and submit the transaction. (If you like to verify the transaction details before signing, you can click on the dropdown button pointed by the arrow in the snapshot below)
+
+![Sign asset creating transaction](../../assets/kusama/statemine-asset-3.png)
+
+If the transaction is successful,  you should see the asset and its details displayed in the Network > Assets page on Statemine.
+
+## Transferring Assets on Statemine
+
+::: caution Beware of scams
+
+Due to the permisionless nature of Statemine, anyone can create an asset with any name and symbol. The most important attribute 
+of an asset is the asset ID. There could be multiple assets with the same name and symbol, but they cannot have the same asset ID. Ex: The USDT by Tether minted on Statemine holds the asset ID: 1984.
+
+:::
+
+To hold, send or receive most of the assets on Statemine, the accounts need to have a minimum balance of 0.1 microKSM (The Existential Deposit on Statemine). You can send KSM from a Kusama account to a statemine account using the teleport functionality. For instructions on teleporting KSM, check this [tutorial on Teleports](../../learn/learn-teleport). The balance transfers of KSM on Statemine are similar to the balance transfers KSM on Kusama and can be done on the accounts page on Polkadot-JS Apps UI.
+
+In this tutorial, transfer of BILLCOINs (Asset ID: 223, Symbol: BILL) is demonstrated.
+
+::: info
+
+Ensure that the receiver account has at least 0.1 microKSM which is the Existential Deposit for a Statemine acocunt
```

Comment:
I would add a note that accounts must exist to hold an asset, unless that asset is `sufficient` OR someone has placed a deposit for that account to hold the asset via `touch`. This is different to Ethereum where a non-existent account can hold ERC-20 tokens, so may confuse some users.

Reply:
Added the note